codewof/programming/content/en/days-to-target/question.md

```diff
@@ -0,0 +1,10 @@
+# Days to Target
+
+A particular population of butterflies doubles in size every day. Write a function `days_to_target(starting_population, target_population)` that takes two numbers:
```

Comment:
Woah, that's a fast growing population of butterflies (just a note, no need to change).

new-sentence-on-new-line